src/system_database.ts

```diff
@@ -1683,10 +1681,41 @@ export class PostgresSystemDatabase implements SystemDatabase {
     return { workflows };
   }
 
-  async enqueueWorkflow(workflowId: string, queueName: string): Promise<void> {
+  async enqueueWorkflowInternal(
```

Comment:
This method should be named `#enqueueWorkflow` instead of `enqueueWorkflowInternal`.